packages/material-ui/src/Grid/Grid.js

```diff
@@ -224,13 +224,11 @@ const Grid = React.forwardRef((props, ref) => {
       [classes.item]: item,
       [classes.zeroMinWidth]: zeroMinWidth,
       [classes[`spacing-xs-${String(spacing)}`]]: container && spacing !== 0,
-      [classes[`direction-xs-${String(direction)}`]]: direction !== Grid.defaultProps.direction,
-      [classes[`wrap-xs-${String(wrap)}`]]: wrap !== Grid.defaultProps.wrap,
-      [classes[`align-items-xs-${String(alignItems)}`]]:
-        alignItems !== Grid.defaultProps.alignItems,
-      [classes[`align-content-xs-${String(alignContent)}`]]:
-        alignContent !== Grid.defaultProps.alignContent,
-      [classes[`justify-xs-${String(justify)}`]]: justify !== Grid.defaultProps.justify,
+      [classes[`direction-xs-${String(direction)}`]]: direction != null,
+      [classes[`wrap-xs-${String(wrap)}`]]: wrap != null,
+      [classes[`align-items-xs-${String(alignItems)}`]]: alignItems != null,
+      [classes[`align-content-xs-${String(alignContent)}`]]: alignContent != null,
+      [classes[`justify-xs-${String(justify)}`]]: justify != null,
```

Comment:
Did you consider doing?

```suggestion
      [classes[`justify-xs-${String(justify)}`]]: justify != 'flex-start',
```